js/platform-window.js

```diff
@@ -53,6 +55,51 @@ class LeftMenu extends HTMLElement {
         }, fin.me.identity);
     }
 
+    async addChartToColumn() {
+        //we want to add a chart to the current window.
+        const layoutApi = await fin.Platform.Layout.getCurrentSync();
+        const winLayoutConfig = await layoutApi.getConfig();
+        const firstLevelColumnIndex = this.getIndexFor("column", winLayoutConfig.content);
+        let insertedChart = false;
+        let newChart = {"type":"stack","isClosable":true,"reorderEnabled":true,"title":"","activeItemIndex":0,"content":[{"type":"component","componentName":"view","componentState":{"name":"component_A1","processAffinity":"ps_1","url":"https://cdn.openfin.co/embed-web/chart.html","componentName":"view","uuid":"platform_customization_local","initialUrl":"https://cdn.openfin.co/embed-web/chart.html"},"isClosable":true,"reorderEnabled":true,"title":"OpenFin Template"}]};
```

Comment:
because this is meant as a tech demo and we want to keep things super readable, can this be a normal JS Object that you stringify if needed ? I feel like it would read easier.

Reply:
Good point.

Tidied it up and fixed a bug where the componentState.name property was not using the componentNameRandomizer function.